incskill_site/templates/quiz1.html

```diff
@@ -522,6 +540,9 @@ <h1>Please Pick An Option</h1>
 //also handles displaying players and quiz information to dom
 function NextQuestion(index) {
     handleQuestions()
+    if(index == 0) {
```

Comment:
```suggestion
    if(index === 0) {
```
best js practice is always to use `===` rather than `==`